### ISSUE TITLE
Lazy Loading

### ISSUE THREAD

**Issue by Inventor77** [avinashkranjan/Recess#142]
**Is your feature request related to a problem? Please describe.**
When Posts in Website will be many so webpage reduce page weight, allowing for a quicker page load time

**Describe the solution you'd like**
This Problem of Page Loading will be solved by Applying Lazy Load on Posts

**Comment by Inventor77** [comment]
@Utkarsh1999 @AnjaliSrivastava722 @Jaagrav @abhishek2x  Can Mentors Please overlook on this and feels if it is worth changing Please assign me this issue and labeling it with gssoc21

**Comment by Jaagrav** [comment]
I'm not a mentor but I do support @Inventor77 
And I myself am interested in solving this although we can do it with react-virtualized, which would make our life a lot easier.

**Comment by Inventor77** [comment]
> I'm not a mentor but I do support @Inventor77 
> And I myself am interested in solving this although we can do it with react-virtualized, which would make our life a lot easier.

Yes Definitely we Both can do It

**Comment by Inventor77** [comment]
Hey @Jaagrav  will it be okay if I start working on this and when mentor allow us I will PR the repo I have

**Comment by Jaagrav** [comment]
I think we should wait for the mentors to merge the remaining PRs since a lot of the code is going to change soon due to the new UI

**Comment by Utkarsh1999** [comment]
hey @Inventor77 , you can start working on this. Also, @Jaagrav , let me know on discord if you both want to collab on this issue.

**Comment by Inventor77** [comment]
> hey @Inventor77 , you can start working on this. Also, @Jaagrav , let me know on discord if you both want to collab on this issue.

Sir First let me Try this and then I will Inform you for help please then only Assign jaagrav with me

**Comment by Utkarsh1999** [comment]
@Inventor77 , any update on this?

**Comment by Inventor77** [comment]
@Utkarsh1999 Sir please lend me some time, I will do this. Im just busy with my College Exams

**Comment by subhangi2731** [comment]
please assign me this. I will make a pr by today

**Comment by Inventor77** [comment]
> please assign me this. I will make a pr by today

@subhangi2731 Hey I'm on it, busy with my college Exams.

**Comment by shobit1337** [comment]
@Utkarsh1999 I guess Inventor is still busy. Since It's been 25 days, Can I work on this issue?

**Comment by Inventor77** [comment]
> @Utkarsh1999 I guess Inventor is still busy. Since It's been 25 days, Can I work on this issue?

@shobit1337 Hello Shobit, Im on it just got little problem with my laptop and you know the lockdown situation so unable repair it. I will definitely do it as soon as possible

**Comment by shobit1337** [comment]
> > @Utkarsh1999 I guess Inventor is still busy. Since It's been 25 days, Can I work on this issue?
> 
> @shobit1337 Hello Shobit, Im on it just got little problem with my laptop and you know the lockdown situation so unable repair it. I will definitely do it as soon as possible
Yeah buddy, But you have already taken a lot of time and already so many issues are assigned to you. So, I think you should ask to assigned to you once your laptop is repaired and you are free from ur exams. Till then let other participants work on these issues 😄

**Comment by Inventor77** [comment]
> > > @Utkarsh1999 I guess Inventor is still busy. Since It's been 25 days, Can I work on this issue?
> > 
> > @shobit1337 Hello Shobit, Im on it just got little problem with my laptop and you know the lockdown situation so unable repair it. I will definitely do it as soon as possible
> >
> Yeah buddy, But you have already taken a lot of time and already so many issues are assigned to you. So, I think you should ask to assigned to you once your laptop is repaired and you are free from ur exams. Till then let other participants work on these issues 😄 
> 

@shobit1337 Please lend me some time I will do it soon

**Comment by Utkarsh1999** [comment]
Hey @Inventor77 , how much more time you need. please specify so that I can reassign this issue to other contributors after that.

Side Note: Open source is more about the community than the number of issues/PR made, so please try to be humble and supportive with each other.

**Comment by Inventor77** [comment]
> Hey @Inventor77 , how much more time you need. please specify so that I can reassign this issue to other contributors after that.
> 
> Side Note: Open source is more about the community than the number of issues/PR made, so please try to be humble and supportive with each other.

@Utkarsh1999 

Thanks Sir, 

I need 3-4 Days more, please let me lend this much time. 
I know how important the community is and thank you for Priceless Advise but the more important for me is my self growth than The PR and Issue assigned to me, due to this Project I got familiar with react framework and Im learning new thing each time I work on any Issue in this Project. 
I hope you understand what I want to Convey and help me out with it 

Stay safe

**Comment by subhangi2731** [comment]
@Utkarsh1999 @avinashkranjan please assign me this issue

**Comment by avinashkranjan** [comment]
Go Ahead @subhangi2731
